packages/components/src/renderer/keyboard.tsx

```diff
@@ -26,9 +26,9 @@ class KeyboardElements extends React.Component<KeyboardElementsProps> {
   }
 
   componentWillUnmount() {
-    if (this.props.append) {
+    if (this.props.append && Keyboard.appendRef.current?.hasChildNodes()) {
       Keyboard.appendRef.current?.removeChild(this.container)
-    } else {
+    } else if (!this.props.append && Keyboard.prependRef.current?.hasChildNodes()) {
```

Comment:
The `"**Cannot display message**"` error was caused by the fact that we tried to remove the container from the ref's children when it wasn't one.